website/guides/sgb_border.md

```diff
@@ -0,0 +1,178 @@
+## Border limitations
+
+An SGB border has:
+
+- 255 tiles + 1 transparent tile (preferably tile #0)
+- 3 palettes of 15 (+ 1 transparent) colors, (up to 45 solid colors total)
+- a 256x224px tilemap (there's a bit more to this, see [notes](#notes))
+
+## Converting borders
+
+With a recent version of [superfamiconv](https://github.com/Optiroc/SuperFamiconv):
+
+```sh
+superfamiconv -v -i input.png -p output.pal -t output.4bpp -P 4 -m output.pct -M snes --color-zero 0000ff -B 4
+```
+
+- `--color-zero` should be the color that your image for transparency, in my case it was blue.
+	* If your image has an alpha channel, it can be set can also be set to `00000000` to use the actual transparent color; however, this may cause some issues.
+- `-v` is optional, for showing details of the conversion process
+- You can add a row of the transparent color at the top of the image to force superfamiconv to make it tile #0, then `incbin "output.pct", 64` to leave out that row.
+- `-P 4` sets the base palette to the 4th one, and **SGB borders use SNES palettes 4, 5, and 6.** as of writing this, this option only works if you built superfamiconv from source.
+
+## Uploading borders
+
+As stated before, the SGB border consists of tile data, picture data, and palette data. These are split across 2-3 packets:
+
+- `CHR_TRN` (`$99`) is used to send 4KiB of tile data.
+
+	* since the border can use up to 8KiB of tiles, bit 0 of the second byte specifies which "half" you're sending
+		- `$99, $00` if the screen is loaded with the first 4KiB of tile data
+		- `$99, $01` if the screen is loaded with the second 4KiB of tile data
+
+- `PCT_TRN` (`$a1`) is used to send the picture and palette data. it also swaps the border, generally a good idea to send it after the tile data[^1]
+
+	* assuming tiles `0`-`255` use VRAM from `$8000` to `$8fff`:
+		- the picture data must be at `$8000`-`$873f` (last 64 bytes are _usually_ offscreen, see [notes](#notes))
+		- palette data must be at `$8800`-`$885f`
+		- everything else is ignored
+			* how you skip putting data at `$8740`-`$87ff` is up to you, I prefer doing separate copies, valen prefers copying tilemap and palette data in one go, with the area between them pa
+
+See also the related Pan Docs entry: [SGB Command Border](https://gbdev.io/pandocs/SGB_Command_Border.html).
+
+[^1]: You can send a `CHR_TRN` up to \~60 frames after the `PCT_TRN` for it to apply to the current border, but not all emulators will emulate this. its fine to just pretend `CHR_TRN`s must go before `PCT_TRN`
+
+## Notes
+
+1. You can set the first row of tiles to your transparent color to force superfamiconv to put the transparent tile as the 1st tile, however you must then exclude 64 bytes of the tilemap (`incbin "border.pct"` -> `incbin "border.pct", 64`)
+
+2. You can use palettes 0-3 and 7, if you really know what you're doing (animated borders yay). You will probably have to edit the border in YY-CHR, as there aren't really any other tools for that.
```

Comment:
What does "if you really know what you're doing" imply?

Reply:
There may be advanced techniques to draw parts of the border with the game palette (0) instead of the palettes reserved for the border (4-6). Currently no converter supports doing this. Would it be worthwhile to make a test ROM to confirm that it's possible?

Reply:
since its not that practical to use the game palette, it should be a low priority

Reply:
Then I think this shouldn't be mentioned in the tutorial, beyond something like "palette 0 is used by the displayed GB screen, palettes 1-3 and 7 are used by the SGB BIOS and will be overwritten when opening various menus".

Mentioning that they *can* be used, but not what the limitations are, is likely to be more confusing.